### DISCUSSION TITLE
List files included with fs.readFileSync in --detailed-report breakdown

### DISCUSSION THREAD

**Discussion by danieltroger** [parcel-bundler/parcel#5639] [Ideas] [answered]
Files included by fs.readFileSync are not listed in --detailed-report, is that something that has to be that way?

It would be kinda nice, because right now they are listed like this:

```
├── Code from unknown sourcefiles                                  98.09 KB      0ms
```

**Comment by mischnic** [comment, marked as answer]
That overview is simply parsing the output sourcemaps. So if we correctly mapped the inlined file to the corresponding source, it should work.

Not sure if there could be downsides to that regarding the sourcemaps itself.

**Comment by danieltroger** [comment]
That would be cool! I guess the question is how it should show up when viewing the sourcemaps in a browser…
